mern/server/models/flow.model 2.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from "mongoose";
```

Comment:
Same comment as withRouter 2, why do we have this new file with no changes?

Reply:
ohhh I realized this is because when I try to restore it to a previous commit it makes the new files, deleted now!